pulpcore/pulpcore/app/viewsets/__init__.py

```diff
@@ -1,10 +1,16 @@
 from .base import AsyncRemoveMixin, AsyncUpdateMixin, BaseFilterSet, NamedModelViewSet  # noqa
+# These need to be imported prior to ContentViewSet - do not shuffle this by alphabetical order
+from .custom_filters import (  # noqa
```

Comment:
I struggled a little bit with circular imports here -- should we just move these to viewsets.content since they're only used in that one place?